packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -11,24 +11,23 @@ import { Meta } from '@storybook/addon-docs';
 
 Before you install the package, make sure that you have performed following steps:
 
-- You must be running Node version >=14.0.0
-- You must have `yarn` installed
```

Comment:
`yarn` is not required for consumers. They can still install using `npm`.

Reply:
Yeah tho better to use `yarn`, previously it was must to use yarn because of blade-old dependency on yarn v1.19.0.

Reply:
Yup in installation commands we're showing yarn so maybe that counts as a recommendation?

Reply:
Works!

Reply:
In my experience using `yarn` has created lesser errors in React Native 😹

Reply:
Lol strange 🤣 

But yeah people will anyway use the package installer that they have configured for that particular project. Don't think so from blade's end we can ask them to use particular installer.